content/source/docs/enterprise/admin/logging.html.md

```diff
@@ -316,4 +316,4 @@ recommend forwarding all Terraform Enterprise logs to a log aggregation system,
 filtering the audit logs based on the `[Audit Log]` string, and forwarding just
 the audit logs to the desired destination.
 
--> **Note:** You can find the previous version of this webpage with logspout information (for releases prior to v202109-1), [here](/docs/enterprise/install/logging-old.html). 
+-> **Note:** You can find the previous version of this webpage with logspout information (for releases prior to v202109-1), [here](/docs/enterprise/admin/logging-old.html). 
```

Comment:
Hello! Just reworking because we want to make sure that we're using link text that's descriptive :-) 

```suggestion
-> **Note:** If you are using a version v202109-1, please refer to the [previous version of this page](/docs/enterprise/admin/logging-old.html) with logspout information.
```

Reply:
the 'logspout' bit is the keyword I'm assuming customers will be looking for. this last release we introduced fluentd instead, but most customers are still on some previous version of the product.

Reply:
```
-> **Note:** If you are using a version of Terraform Enterprise prior to v202109-1, please refer to the [previous version of this page](/docs/enterprise/admin/logging-old.html) with logspout information.
```